src/V3Param.cpp

```diff
@@ -1234,7 +1240,8 @@ class ParamVisitor final : public VNVisitor {
     }
     void visit(AstParamTypeDType* nodep) override {
         iterateChildren(nodep);
-        if (VN_IS(nodep->skipRefp(), VoidDType)) {
+        const AstNodeDType* skipRefp = nodep->skipRefOrNullp();
+        if (skipRefp && VN_IS(skipRefp, VoidDType)) {
```

Comment:
```suggestion
        if (VN_IS(nodep->skipRefOrNullp(), VoidDType)) {
```

VN_IS already returns false for nullptr